pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -894,7 +895,7 @@ func (l *LibvirtDomainManager) SyncVMI(vmi *v1.VirtualMachineInstance, useEmulat
 
 	// Resize and notify the VM about changed disks
 	for _, disk := range domain.Spec.Devices.Disks {
-		if imageCanBeExpanded(disk) {
+		if disk.ForceResize || imageCanBeExpanded(disk) {
```

Comment:
Perhaps the check for `disk.ForceResize` should be pushed down into `imageCanBeExpanded()`.

The flag itself should probably be named `AlreadyExpanded` or something along those lines.

Reply:
Went for AlwaysExpand.
Having "ShouldExpand(disk)" always return true for "AlreadyExpanded" seemed slightly off.